stl/inc/ranges

```diff
@@ -40,6 +41,11 @@ namespace ranges {
 
     template <class _Ty>
     concept _Copy_constructible_object = copy_constructible<_Ty> && is_object_v<_Ty>;
+
+    template <class _It>

```

Comment:
I'd like to:
* leave this name alone (this is _`has-arrow`_ from [[range.utility.helpers]](http://eel.is/c++draft/range.utility.helpers), I want to keep the name as close as possible but `_Ugly` instead of _`kebab-case`_),
* rename the other concept to the more precise `_Has_member_arrow`, and
* use `_Has_member_arrow<_It&>` to replace the *requires-expression* in the definition of this concept.

Does that sound reasonable?

Reply:
Absolutely